examples/pgbouncer/README.md

```diff
@@ -0,0 +1,260 @@
+# Using the Cloud SQL Auth Proxy with PgBouncer
+
+This example demonstrates how to run the Cloud SQL Auth Proxy with PgBouncer on
+Kubernetes. It assumes you have already successfully completed all the steps in
+[Using the Cloud SQL Auth Proxy on Kubernetes][k8s-example].
+
+In this example, you will deploy [PgBouncer][] with the proxy as a sidecar, in
+addition to configuring encryption between the application and PgBouncer.
+
+## Warning
+
+This is an advanced example. Running PgBouncer may be a significant operational
+burden. Nonetheless, running the proxy with PgBouncer may be an appropriate
+choice when running the proxy as a sidecar no longer works effectively at your
+scale.
+
+## Generate Certificates for PgBouncer
```

Comment:
I would consider moving the following steps as substeps under `## Initial Setup`:
1. Generate Certificates for PgBouncer
2. Save the certificates as secrets
3. Containerize PgBouncer

Reply:
Done.